components/ui/Footer.js

```diff
@@ -236,7 +236,7 @@ const Footer = () => (
       </Box>
       <Box my={mediaQueryLessThan('0px', 'mobileLarge', '8px')}>
         <Text color={SUBTITLE_COLOR} letterSpacing={0.3}>
-          {'Last Updated — '}<time dateTime="2021-03">{'March, 2021'}</time>
+          {'Last Updated — '}<time dateTime="2021-05">{'May, 2021'}</time>
```

Comment:
Made sure to update the footer here, but I shouldn't have to remember this. We can probably automate this value here